ui/app/pages/permissions-connect/choose-account/index.scss

```diff
@@ -9,26 +9,76 @@
     @extend %header--18;
   }
 
-  &__text {
+  &__text, &__text--blue, &__text--grey {
     @extend %content-text;
     line-height: 25px;
   }
 
+  &__text-blue {
+    color: $curious-blue;
+    cursor: pointer;
+  }
+
+  &__text-grey {
+    color: $Grey-500;
+  }
+
   &__accounts-list {
-    width: 393px;
+    width: 428px;
     border: 1px solid #D0D5DA;
     box-sizing: border-box;
     border-radius: 8px;
-    margin-top: 36px;
+    margin-top: 8px;
+    max-height: 338px;
+    overflow-y: scroll;
```

Comment:
Nit: We shouldn't display a scrollbar here unless it's required.

The scrollbar also doesn't match the design of the scrollbars used elsewhere in the extension.

Reply:
fixed in 532b5bf4d